### DISCUSSION TITLE
DietPi Software | Portainer Agent

### DISCUSSION THREAD

**Discussion by holocronology** [MichaIng/DietPi#6225] [Software Request]
Installation of portainer agent only, as opposed to portainer, for those using remote portainer management.

https://docs.portainer.io/start/agent/docker/linux

**Comment by Joulinar** [comment]
Please vote for this software implementation via ⬆️ button at first post.

**Comment by iodize6399** [comment]
+1

Here's the correct URL: https://docs.portainer.io/admin/environments/add/docker/agent

I would also like to point out that this agent is different from edge agent https://docs.portainer.io/admin/environments/add/docker/edge

**Comment by pedrom34** [comment]
That would be a great addition to control docker via portainer on multiple machines

**Comment by Dubz** [comment]
With as much stuff jam packed into DietPi and people looking to create clusters, I'm genuinely surprised this has to be requested specifically and wasn't just done from the start.

+1

**Reply by AlexFigas** [reply]
Agreed

**Comment by AlexFigas** [comment]
+1

**Comment by mews-se** [comment]
I seem to recall that portainer actually was included in the software list a while back.

Since there are two versions, ce and be it would be nice to be able to choose which to install!

**Reply by MichaIng** [reply]
Our implementation is the free community edition. However, not related to Portainer Agent.
